eng/common/scripts/Add-RetentionLease.ps1

```diff
@@ -19,14 +19,19 @@ param(
   [string]$OwnerId = "azure-sdk-pipeline-automation",
 
   [Parameter(Mandatory = $false)]
-  [string]$AccessToken = $env:DEVOPS_PAT
```

Comment:
Do we even need to change this script if we don't have any plans to use the bear token for it? I think it is fine to leave this script unchanged.

Reply:
I removed the Base64EncodedAuthToken script parameter which was never used. I also removed the BearerToken parameter considering that this script is only ever going to get called using the System.AccessToken for the pipeline the retain-run.yml file is used in.